### PR TITLE
[PPP-4865] - Vulnerable Component: snakeyaml

### PR DESCRIPTION
**Pull request by andreramos89**
@smmribeiro 
@renato-s

### CODE REVIEW

**Comment by buildguy** [comment]
### :x: Build failed in 30s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 29s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

File: core/pom.xml [hunk 1]
```diff
@@ -525,9 +525,9 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.yaml</groupId>
-      <artifactId>snakeyaml</artifactId>
-      <scope>test</scope>
+      <groupId>org.hitachivantara</groupId>
+      <artifactId>safeyaml</artifactId>
+      <version>${safeyaml.version}</version>
```
Comment (smmribeiro) [inline core/pom.xml:530]: Shouldn't this be test-scoped?

**Review by smmribeiro — CHANGES_REQUESTED** [review]

**Comment by buildguy** [comment]
### :x: Build failed in 30s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Review by smmribeiro — APPROVED** [review]
Looks good.

**Pull request merged**